### PR TITLE
ko: cr2 Android translations

### PR DESCRIPTION
**Pull request by henrif75**
#262 #1463
*Stats*
Before: 1297 translated messages, 383 fuzzy translations, 1780 untranslated messages.
After: 1420 translated messages, 474 fuzzy translations, 1566 untranslated messages.

### CODE REVIEW

**Comment by henrif75** [comment]
Stats after clean-up:
1419 translated messages, 426 fuzzy translations, 1615 untranslated messages.

File: po/ko.po [hunk 1]
```diff
@@ -11337,16 +11347,21 @@ msgstr ""
 "setup/start)을 참조하십시오."
 
 #: src/android/setup.md:20
+#, fuzzy
 msgid ""
 "Cuttlefish is a reference Android device designed to work on generic Linux "
 "desktops. MacOS support is also planned."
 msgstr ""
+"Cuttlefish는 제네릭 Linux 데스크톱에서 작동하도록 설계된 참조 Android 기기입"
```
Comment (jiyongp) [inline po/ko.po:11355]:
```suggestion
"Cuttlefish는 일반적인 Linux 데스크톱에서 작동하도록 설계된 참조 Android 기기입"
```

File: po/ko.po [hunk 2]
```diff
@@ -11337,16 +11347,21 @@ msgstr ""
...
+"니다.  MacOS 지원도 계획되어 있습니다."
 
 #: src/android/setup.md:23
 msgid ""
 "The Cuttlefish system image maintains high fidelity to real devices, and is "
 "the ideal emulator to run many Rust use cases."
 msgstr ""
+"Cuttlefish 시스템 이미지는 실제 기기에 높은 충실도를 유지하며, 다양한 Rust 사"
+"용 사례를 실행하는 데 이상적인 에뮬레이터입니다."
```
Comment (jiyongp) [inline po/ko.po:11364]:
```suggestion
"Cuttlefish는 실제 하드웨어를 매우 충실히 재현하고 있으며, Rust 를 사용해 보기에"
"이상적인 에뮬레이터입니다."
```

File: po/ko.po [hunk 3]
```diff
@@ -11444,25 +11459,32 @@ msgstr "다음은 `rust_binary`와 `rust_library`를 살펴봅니다."
...
+"다."
 
 #: src/android/build-rules.md:25
+#, fuzzy
 msgid ""
 "For compliance and performance, Android must have crates in-tree. It must "
 "also interop with C/C++/Java code. Soong fills that gap."
 msgstr ""
+"규정 준수와 성능을 위해 Android는 트리 내에 크레이트가 있어야 합니다.  또한 "
+"C/C++/Java 코드와 상호 운용되어야 합니다.  Soong이 이 공백을 메웁니다."
```
Comment (jiyongp) [inline po/ko.po:11479]:
```suggestion
"Android에서는 규정상, 그리고 빌드 속도를 위해, 크레이트들이 Android 소스코드 트리 안에 미리 포함되어 있어야 합니다. "
"빌드 시 다운로드 받을 수 없습니다.  또한 "
"C/C++/Java 코드와 상호 운용되어야 합니다. Android 빌드 시스템인 Soong이 이 공백을 메웁니다."
```

File: po/ko.po [hunk 4]
```diff
@@ -11444,25 +11459,32 @@ msgstr "다음은 `rust_binary`와 `rust_library`를 살펴봅니다."
...
 
 #: src/android/build-rules.md:28
 msgid ""
 "Soong has many similarities to Bazel, which is the open-source variant of "
 "Blaze (used in google3)."
 msgstr ""
+"Soong은 Blaze(google3에서 사용됨)의 오픈소스 변형인 Bazel과 많은 유사점이 있"
+"습니다."
```
Comment (jiyongp) [inline po/ko.po:11487]:
```suggestion
"Soong은 Blaze(google3에서 사용됨)의 오픈소스 변형인 Bazel과 많이 비슷합니다."
```

File: po/ko.po [hunk 5]
```diff
@@ -11471,14 +11493,19 @@ msgid ""
 "com/chromiumos/bazel/), and [Fuchsia](https://source.android.com/docs/setup/"
 "build/bazel/introduction) to Bazel."
 msgstr ""
+"[Android](https://source.android.com/docs/setup/build/bazel/introduction), "
+"[ChromeOS](https://chromium.googlesource.com/chromiumos/bazel/), [Fuchsia]"
+"(https://source.android.com/docs/setup/build/bazel/introduction)를 Bazel로 전"
+"환할 계획입니다."
```
Comment (jiyongp) [inline po/ko.po:11499]: Android stopped the transition to Bazel a few months ago. We may have to amend the original text.

File: po/ko.po [hunk 6]
```diff
@@ -11471,14 +11493,19 @@ msgid ""
...
 
 #: src/android/build-rules.md:37
 msgid "Learning Bazel-like build rules is useful for all Rust OS developers."
 msgstr ""
+"Bazel과 유사한 빌드 규칙을 배우는 것은 모든 Rust OS 개발자에게 유용합니다."
 
 #: src/android/build-rules.md:39
 msgid "Fun fact: Data from Star Trek is a Soong-type Android."
-msgstr ""
+msgstr "재미있는 사실: 스타트렉의 데이터는 Soong 타입의 Android입니다."
```
Comment (jiyongp) [inline po/ko.po:11508]:
```suggestion
msgstr "재미있는 사실: 스타트렉의 캐릭터 중 한명인 데이터(Data)는 사실 Soong 타입의 안드로이드(Android)입니다."
```

File: po/ko.po [hunk 7]
```diff
@@ -11497,30 +11524,32 @@ msgid "_hello_rust/Android.bp_:"
...
 msgid "\"hello_rust\""
-msgstr ""
+msgstr "\"hello_rust\""
 
 #: src/android/build-rules/binary.md:12 src/android/build-rules/library.md:19
 #: src/android/logging.md:12
 #, fuzzy
 msgid "\"src/main.rs\""
-msgstr "_hello_rust/src/main.rs_:"
+msgstr "\"main.c\""
```
Comment (jiyongp) [inline po/ko.po:11535]:
```suggestion
msgstr "\"src/main.c\""
```

File: po/ko.po [hunk 8]
```diff
@@ -11569,46 +11598,47 @@ msgstr ""
...
 msgid "\"greetings\""
-msgstr ""
+msgstr "\"인사말\""
 
 #: src/android/build-rules/library.md:30 src/android/aidl/implementation.md:29
 #: src/android/interoperability/java.md:39
 #, fuzzy
 msgid "\"src/lib.rs\""
-msgstr "_hello_rust/src/main.rs_:"
+msgstr "\"lib.rs\""
```
Comment (jiyongp) [inline po/ko.po:11625]:
```suggestion
msgstr "\"src/lib.rs\""
```

File: po/ko.po [hunk 9]
```diff
@@ -11830,30 +11863,32 @@ msgid "_birthday_service/src/client.rs_:"
...
 #, fuzzy
 msgid "\"birthday_client\""
-msgstr "_birthday_service/src/client.rs_:"
+msgstr "\"libbirthday_bindgen\""
 
 #: src/android/aidl/client.md:42
 #, fuzzy
 msgid "\"src/client.rs\""
-msgstr "_hello_rust/src/main.rs_:"
+msgstr "\"lib.rs\""
```
Comment (jiyongp) [inline po/ko.po:11891]:
```suggestion
msgstr "\"src/lib.rs\""
```

File: po/ko.po [hunk 10]
```diff
@@ -11904,43 +11939,46 @@ msgstr "_hello_rust_logs/Android.bp_:"
 #: src/android/logging.md:10 src/android/logging.md:11
 #, fuzzy
 msgid "\"hello_rust_logs\""
-msgstr "_hello_rust_logs/Android.bp_:"
+msgstr "\"hello_rust\""
```
Comment (jiyongp) [inline po/ko.po:11942]:
```suggestion
msgstr "\"hello_rust_logs\""
```

File: po/ko.po [hunk 11]
```diff
@@ -12345,35 +12398,37 @@ msgstr ""
 #: src/chromium/interoperability-with-cpp/error-handling-png.md:9
 #, fuzzy
 msgid "\"Rust\""
-msgstr "Rustdoc"
+msgstr "\"rust\""
```
Comment (jiyongp) [inline po/ko.po:12401]:
```suggestion
msgstr "\"Rust\""
```

File: po/ko.po [hunk 12]
```diff
@@ -12322,19 +12372,22 @@ msgid ""
...
 #: src/android/interoperability/cpp/bridge.md:11
 msgid "// Shared structs with fields visible to both languages.\n"
-msgstr ""
+msgstr "// 두 언어 모두에 표시되는 필드가 있는 공유 구조체입니다.\n"
 
 #: src/android/interoperability/cpp/bridge.md:17
 #: src/android/interoperability/cpp/generated-cpp.md:6
 msgid "// Rust types and signatures exposed to C++.\n"
-msgstr ""
+msgstr "// C++에 노출된 Rust 타입 및 서명입니다.\n"
```
Comment (jiyongp) [inline po/ko.po:12390]:
```suggestion
msgstr "// C++에 노출된 Rust 타입 및 메서드 시그니쳐입니다.\n"
```

File: po/ko.po [hunk 13]
```diff
@@ -12345,35 +12398,37 @@ msgstr ""
...
 #, fuzzy
 msgid "\"Rust\""
-msgstr "Rustdoc"
+msgstr "\"rust\""
 
 #: src/android/interoperability/cpp/bridge.md:24
 #: src/android/interoperability/cpp/cpp-bridge.md:6
 msgid "// C++ types and signatures exposed to Rust.\n"
-msgstr ""
+msgstr "// Rust에 노출된 C++ 타입 및 서명입니다.\n"
```
Comment (jiyongp) [inline po/ko.po:12406]:
```suggestion
msgstr "// Rust에 노출된 C++ 타입 및 함수 시그니쳐입니다.\n"
```

File: po/ko.po [hunk 14]
```diff
@@ -12417,47 +12479,54 @@ msgid ""
...
+msgstr "\"org$blobstore$cxxbridge1$BlobstoreClient$put\""
 
 #: src/android/interoperability/cpp/cpp-bridge.md:56
 msgid ""
 "The programmer does not need to promise that the signatures they have typed "
 "in are accurate. CXX performs static assertions that the signatures exactly "
 "correspond with what is declared in C++."
 msgstr ""
+"프로그래머는 자신이 입력한 서명이 정확하다고 보장할 필요가 없습니다. CXX는 서"
+"명이 C++에서 선언된 것과 정확히 일치한다는 정적 어설션을 실행합니다."
```
Comment (jiyongp) [inline po/ko.po:12513]:
```suggestion
"프로그래머는 자신이 입력한 시그니쳐가 정확하다고 보장할 필요가 없습니다. CXX는 시그니쳐가"
"C++에서 선언된 것과 정확히 일치하는지를 체크하기 위해 정적으로 assertion을 실행합니다."
```

File: po/ko.po [hunk 15]
```diff
@@ -12466,15 +12535,19 @@ msgid ""
...
 msgstr ""
+"공유 타입의 `#[derive()]`에는 제한된 수의 트레잇이 지원됩니다. C++ 코드에 대"
+"해서도 상응하는 기능이 생성됩니다. 예를 들어 `Hash`를 파생하면 해당 C++ 타입"
+"에 대한 `std::hash` 구현도 생성됩니다."
 
 #: src/android/interoperability/cpp/shared-enums.md:15
 #, fuzzy
 msgid "Generated Rust:"
-msgstr "안전하지 않은 러스트"
+msgstr "생성된 C++:"
```
Comment (jiyongp) [inline po/ko.po:12545]:
```suggestion
msgstr "생성된 러스트:"
```

**Review by jiyongp — COMMENTED** [review]
I've left some inline suggestions.

File: po/ko.po (hunk 5 above)
Comment (henrif75) [reply po/ko.po:11499]: Created an issue to track this.

**Review by jiyongp — APPROVED** [review]

**Comment by henrif75** [comment]
Thanks @jiyongp !

**Pull request merged**